### DISCUSSION TITLE
How to mark a discussion as answered?

### DISCUSSION THREAD

**Discussion by pllim** [ejeschke/ginga#927] [Q&A] [answered]
GitHub says it is possible at https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions#marking-a-comment-as-an-answer but I cannot see that button. @ejeschke , can you see it?

**Comment by ejeschke** [comment, marked as answer]
Hmm, well I don't see it in the other threads, but in this thread I see this:
![answer_button](https://user-images.githubusercontent.com/1305473/106648761-c9022d80-6534-11eb-920a-0f788e5937fb.png)

**Comment by ejeschke** [comment]
So I guess you don't see it for your own posts?

**Reply by ejeschke** [reply]
Do you see it now on this thread?

**Reply by pllim** [reply]
Ah, weird. Now I see it for this post. But I don't see it for the other ones like https://github.com/ejeschke/ginga/discussions/922 .

So, only "Q&A" types have it? That is annoying, as I don't see a way to close them like Issues either.

**Reply by pllim** [reply]
Looks like I have to contact GitHub help desk again...

**Comment by ejeschke** [comment]
Seems like they are trying to take a page from Stack Overflow...

**Comment by ejeschke** [comment]
Maybe it's doing some kind of AI on the text to decide whether to render with an "Answer" or "Comment" button?

**Comment by ejeschke** [comment]
@pllim, how to you feel about deleting discussions?  I see a "delete discussion" link in the right-hand sidebar.  Might make sense for topical, or time-sensitive discussions that are no longer relevant...

**Reply by pllim** [reply]
I guess we can do that. Wish it has a "Close" like issues though.

**Comment by pllim** [comment]
@ejeschke , apparently we can also change the discussion category after-the-fact. So I guess if we really want to, we can convert everything to Q&A and then select the answer? 🤷 

![Screen Shot 2021-02-04 at 4 15 01 PM](https://user-images.githubusercontent.com/2090236/106979403-404de200-672c-11eb-8e03-31d32536fea1.png)

**Reply by ejeschke** [reply]
Ah, so there is your info! Only threads in the Q&A category can be "answered".
